player/command.c

```diff
@@ -507,7 +507,10 @@ static int mp_property_path(void *ctx, struct m_property *prop,
     MPContext *mpctx = ctx;
     if (!mpctx->filename)
         return M_PROPERTY_UNAVAILABLE;
-    return m_property_strdup_ro(action, arg, mpctx->filename);
+    char *path = mp_normalize_path(mpctx, mpctx->filename);
```

Comment:
use temporary talloc_ctx, it will leave some allocations that won't be deallocated if attached to mpctx